engine/render/src/render/render.cpp

```diff
@@ -557,6 +640,25 @@ namespace dmRender
             SortRenderList(context);
         }
 
+        dmhash_t frustum_hash = frustum_matrix ? dmHashBuffer64((const void*)frustum_matrix, 16*sizeof(float)) : 0;
```

Comment:
We want to avoid recalculating the visibility unless the frustum actually changes.